vite.config.ts

```diff
@@ -7,4 +7,10 @@ export default defineConfig({
       ignoreConfigErrors: true,
     }),
   ],
+  test: {
+    maxConcurrency: 10,
+    // configuration to be able to view console.log messages while debugging
+    pool: "forks",
+    disableConsoleIntercept: true,
```

Comment:
```suggestion
    disableConsoleIntercept: Boolean(process.env.CI)
```

without checking myself, I suspect this might be it to make Github Actions happy, and/or the `pool` option